### DISCUSSION TITLE
HZ-PS01 Human Presence detector

### DISCUSSION THREAD

**Discussion by Kaveendra** [Koenkk/zigbee2mqtt#23424] [Devices]
I have bought this device, without much thought and found out it is detected but unrecognized by Zigbee2MQTT! 
https://www.aliexpress.us/item/3256806851162604.html

However, it is detected as model TS0601 on Z2M device page, and that model supposedly a supported device on Z2M!

Has anybody come-across this, or found a solution please

**Comment by Kaveendra** [comment]
![Tuya-Zigbee-Presence-Sensor](https://github.com/user-attachments/assets/75d7783a-87dd-42ef-ada6-f307c2c71c1e)
<img width="277" alt="Screenshot 2024-07-26 123240" src="https://github.com/user-attachments/assets/8abfe2cb-a1c3-4757-9d8a-ca5142422c0f">

**Comment by rensjbeekhuizen** [comment]
i have the same issues here, but with deconz and ZHA.

**Comment by Koenkk** [comment]
`_TZE204_nbkshs6k` will be supported in tomorrows Z2M release.

**Reply by Kaveendra** [reply]
Thanks mate.
I have checked this morning and happy to report it's working great!
